ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -970,6 +968,9 @@ def _on_span_finished(span: Span, streamed_chunks):
         else:
             # best effort to join chunks together
             content = "".join([str(chunk) for chunk in streamed_chunks])
+        integration.llmobs_set_tags(span, args=args, kwargs=kwargs, response=content, operation="chain")
+        if span.error or not integration.is_pc_sampled_span(span):
```

Comment:
Do we want to set llm obs tags before we do the `integration.is_pc_sampled_span`?

Reply:
good question. it looks like we recently changed `llmobs_set_tags` to run independently of `is_pc_sampled_span` ([example](https://github.com/DataDog/dd-trace-py/blob/d490d8ec05dffe06fabaeff98cd4b9ed0483f514/ddtrace/contrib/internal/langchain/patch.py#L250)). followed the same logic here, but i think even before we did something like `is_pc_sampled_llmobs`, which I think always returned `True`.